src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,377 @@
+# Functional Near-Infrared Spectroscopy
+
+Support for functional Near-Infrared Spectroscopy (fNIRS) was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+## fNIRS recording data
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["nirs", "events"]) }}
+
+Only the Shared Near Infrared Spectroscopy Format ([SNIRF](https://github.com/fNIRS/snirf))
+file specification is supported in BIDS. The SNIRF
+specification supports one or more fNIRS datasets to be stored in a single
+`.snirf` file. However, to be BIDS compatible, each SNIRF file MUST contain
+only a single run. A limited set of fields from the SNIRF specification are
+replicated  in the BIDS specification. This redundancy allows the data to be
+easily parsed by humans and machines that do not have a SNIRF reader at hand,
+which improves findability and tooling development.
+
```

Comment:
```suggestion
Template: 

sub-<label>/
    [ses-<label>/]
        nirs/
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_nirs.snirf
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_nirs.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_nirs.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_channels.tsv
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_optodes.tsv
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>]_coordsystem.json

```

We usually present a template with the data organization at the beguining

Reply:
Doesn't the macro in the lines above create this?